src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+:root {
+  --main-font: "Avenir", "Roboto", sans-serif;
+  --main-color: #3d4e61;
+  --border-color: #e1e7ed;
+  --shadow-color: #3d4e611a;
+  --hover-color: #3d4e6133;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: var(--main-font);
+}
```

Comment:
❌ Never reset margin for ALL elements. `*` should be used for `box-sizing` property